drivers/Makefile.dep

```diff
@@ -70,6 +70,18 @@ ifneq (,$(filter bh1750fvi,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter bme680_%,$(USEMODULE)))
+  USEPKG += bme680_driver
+  USEMODULE += bme680
+endif
+ifneq (,$(filter bme680_i2c, $(USEMODULE)))
```

Comment:
I'm not sure if this is a coding convention, but we always use an empty line between successive `ifneq ... endif' statements, even if these statements are related.